modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemander.java

```diff
@@ -455,6 +438,13 @@ private void requestPartitions(final RebalanceFuture fut, GridDhtPreloaderAssign
             return;
         }
 
+        if (fut.isDone()) {
+            assert !fut.result() : "Rebalance future was done, but partitions never requested grp = "
```

Comment:
Grammar and code style are bad.

Reply:
Could not find grammar mistakes here, but corrected output information.